test/test_images/hellohttp2noh2c/hellohttp2.go

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
This test image is same with https://github.com/knative/serving/blob/master/test/test_images/hellohttp2/hellohttp2.go ? If  yes, let's use the existing one.

Reply:
Oh! Yes, sorry... I was planning to just create an extra yaml file in this directory, and then forgot, will do in a moment. Thank you!